package.json

```diff
@@ -78,6 +80,8 @@
     "@types/react": "17.0.21"
   },
   "peerDependencies": {
+    "@pagopa/io-react-native-jwt": "*",
+    "cbor-x": "*",
```

Comment:
Why not just a dependency?

Reply:
Because for RN-based packages it is necessary to also install the dependencies in the app that uses them otherwise the native part is not imported correctly (for example on pods).

Refs:[ react-native-builder-bob ](https://callstack.github.io/react-native-builder-bob/faq#how-do-i-add-a-react-native-library-containing-native-code-as-a-dependency-in-my-library)